From bcd85897c2e55d955c325f72791d3ff99eacf820 Mon Sep 17 00:00:00 2001
From: James Youngman <jay@gnu.org>
Date: Wed, 30 Dec 2015 20:23:22 +0000
Subject: [PATCH 011/224] Remove the --enable-id-cache configure option.

* configure.ac: Remove the --enable-id-cache option.  This option
became a no-op in findutils-4.5.15.  Before that it did something
but the code was buggy.  See https://savannah.gnu.org/bugs/?45062
for details.
* NEWS: State that this configure option is removed.
---
 NEWS         | 5 +++++
 configure.ac | 3 ---
 2 files changed, 5 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 0b9ef281..bfc94f1c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -15,9 +15,6 @@ AC_SUBST(AUXDIR,$ac_aux_dir)
 dnl check for --with-fts
 FIND_WITH_FTS
 
-AC_ARG_ENABLE(id-cache,
-[  --enable-id-cache       This currently has no effect.],)
-
 AC_ARG_ENABLE(debug,
 	AS_HELP_STRING(--enable-debug,Enable debugging output which is likely to be interesting to people debugging findutils),
 	[ac_cv_debug=$enableval],[ac_cv_debug=no])
-- 
2.19.1

